### PR TITLE
fix(utility): nullable formatting

### PR DESCRIPTION
**Pull request by georgettica**
same as #13793 

related to #13791

### CODE REVIEW

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-wide/Format-Wide.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public SwitchParameter AutoSize
             get
             {
                 if (_autosize.HasValue)
-                    return _autosize.Value;
+                    return _autosize;
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-wide/Format-Wide.cs:53]: We can not assign nullable type to non-nullable one.
Below too.
```suggestion
                    return _autosize.GetValueOrDefault();
```

**Comment by georgettica** [comment]
Conversion issues here aswell, will be fixed tomorrow hopefully

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-wide/Format-Wide.cs (hunk 1 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-wide/Format-Wide.cs]: You should do the same for rest of changes.

**Comment by georgettica** [comment]
cancelling as I think someone else is doing these

**Pull request closed without merging**